### ISSUE TITLE
Upgrade from v22.02.2 to 22.04.2 

### ISSUE THREAD

**Issue by davecowley** [BookStackApp/BookStack#3489]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

Running Ubuntu 18.04.6 LTS

When running the update there are failures.  The 1st pull is fine but when composer install --no-dev is ran it errors and the text is highlighted.  The error messages scrolls and scrolls and I can't go up to see the initial fault.

Any ideas please?

### Exact BookStack Version

v22.02.2

### Log Content

[laravel.log](https://github.com/BookStackApp/BookStack/files/8871214/laravel.log)

### PHP Version

74.28

### Hosting Environment

Ubuntu 18.04.6 LTS
Installed using script

**Comment by ssddanbrown** [comment]
Hi @davecowley,

Can you output the results of running both of the following commands?:

```bash
composer -V
php -v
```

To capture the output of the composer step, you may be able to run it and instead output command line text to a file instead like so:

```bash
composer install --no-dev > log.txt 2>&1
```

Any error/console output should then be captured in `log.txt`.

**Comment by davecowley** [comment]
Hi, heres the info.

Composer 1.6.3
PHP 7.4.28

I ran the command for the composer and it originally failed with permission denied, i think it was the output of the location so I changed it to the home folder.

So, doing all this it actually upgraded.  I dont know why as I did the exact same thing other than the log output.

bizarre.

If you still do want the log then let me know and I'll send it your way. 

Thanks for your help.
